app/services/riiif/imagemagick_command_factory.rb

```diff
@@ -64,9 +68,15 @@ def input
         " #{path}#{layer_spec}"
       end
 
+      # In cases where the input file has an alpha_channel but the transformation
+      #   format is 'jpg', change to 'png' as jpeg does not support alpha channels
       # pipe the output to STDOUT
       def output
-        " #{transformation.format}:-"
+        if alpha_channel? && jpeg?
+          " png:-"
```

Comment:
Can you add a test for this case?

Reply:
Done. Thanks for the quick response. Would you be OK to merge?